SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -82,19 +104,69 @@ type UnrealResponse {
     string some_response_string = 1;
 }
 
-component NonUnrealAuthoritative {
+component TestComponent {
     id = 1337;
     uint32 counter = 1;
+    command UnrealResponse test_command(UnrealRequest);
 }
 
-component UnrealAuthoritative {
+component OtherTestComponent {
     id = 1338;
     uint32 other_counter = 1;
-    command UnrealResponse test_command(UnrealRequest);
 }
 ```
 
-#### Send data
+#### Using the code generator example
+
+Using the above schema file placed in the project relative directory `spatial\schema\improbable\`, the command below will output code to a new `Game\Source\<your_project_name>\ExternalSchemaCodegen` folder:
+
+`Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\ExternalSchemaCodegen.bat spatial\schema\my_external_schema Game\Source\<your_project_name>\ExternalSchemaCodegen`
+
+This will generate code in the `Game\Source\<your_project_name>\ExternalSchemaCode` folder. To see these changes, the Visual Studio solution needs to be regenerated.
```

Comment:
```suggestion
This will generate code in the `Game\Source\<your_project_name>\ExternalSchemaCode` folder. To see these changes in Visual Studio, the Visual Studio project files need to be regenerated.
```